hpcgap/lib/oper.g

```diff
@@ -1276,6 +1276,78 @@ BIND_GLOBAL( "DeclareAttribute", function ( arg )
 end );
 
 
+##############################################################################
+##
+##  This is a temporary (I hope!) hack for HPCGAP, which introduces thread-local
+##  attributes. These are not complete replacements for normal attributes.
+##  In particular, they are not filters, so cannot be used in method selection.
+##
+##  All thread locals are mutable.
```

Comment:
Then why is there a `mutflag` argument? Also, this arg is ignored by HPC-GAP, but honored by GAP, which seems to invite differences between the two.